BlueBrain/HighFive
# Add code coverage through codecov.io

**Description**

This PR introduces code coverage monitoring into the CI. Reports are provided by codecov.io


**How to test this?**

Check github actions on `ci_test` branch. You can also run locally the commands in `.github/workflows/ci.yml`.
